engine/api/src/com/cloud/vm/VirtualMachineManager.java

```diff
@@ -49,12 +49,15 @@
 public interface VirtualMachineManager extends Manager {
 
     static final ConfigKey<Boolean> ExecuteInSequence = new ConfigKey<Boolean>("Advanced", Boolean.class, "execute.in.sequence.hypervisor.commands", "false",
-            "If set to true, StartCommand, StopCommand, CopyCommand, MigrateCommand will be synchronized on the agent side."
+            "If set to true, StartCommand, StopCommand, CopyCommand, Migrateommand will be synchronized on the agent side."
                     + " If set to false, these commands become asynchronous. Default value is false.", false);
```

Comment:
@kansal Could you please correct the "Migrateommand" to "MigrateCommand"? Thanks.